contentwarehouse/snippets/update_document_sample.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_update_document]
+
+from google.cloud import contentwarehouse
+
+
+def sample_update_document(
+    document_name: str, document: contentwarehouse.Document, user_id: str
+) -> contentwarehouse.CreateDocumentResponse:
+    """Updates a document.
+
+    Args:
+        document_name: The resource name of the document.
+                    Format: 'projects/{project_number}/
+                    locations/{location}/documents/{document_id}'.
+        document: Document AI Warehouse Document object..
+        user_id: user_id: user:YOUR_SERVICE_ACCOUNT_ID or user:USER_EMAIL.
+    Returns:
+        Response object.
+    """
+    # Create a client
+    client = contentwarehouse.DocumentServiceClient()
+
+    document.display_name = "Updated Order Invoice"
```

Comment:
Maybe add a comment to say `# Update document fields` Could also give examples of other fields to update?
Is there a doc link you can add that says what fields can be updated?

Reply:
I couldn't found a specific link, adding https://cloud.google.com/document-warehouse/docs/reference/rest/v1/Document

Reply:
That works